src/lib/airtable/tables/requests.js

```diff
@@ -67,7 +67,7 @@ exports.findRequestByExternalId = async (externalId) => {
   }
 };
 
-// Find open requests that area already in Slack
+/** Finds all requests in an open state (ie needing delivery) in Airtable */
 exports.findDeliveryNeededRequests = async () => {
   const requestOpenStates = [
     fields.status_options.dispatchStarted,
```

Comment:
Change requested: Now that we are only looking at requests in Delivery Needed state, we should remove `dispatchStarted` as an open state. Dispatch started requests may not have completed data for the request.